packages/sync-repo-settings/src/bot.ts

```diff
@@ -123,6 +123,48 @@ export function handler(app: Probot) {
     }
   );
 
+  /**
+   * On pushes to the default branch, check to see if the
+   * .github/sync-repo-settings.yaml file was included.
+   * If so, run the settings sync for that repo.
+   */
+  app.on('push', async context => {
+    const branch = context.payload.ref;
+    const defaultBranch = context.payload.repository.default_branch;
+    if (branch !== `refs/head/${defaultBranch}`) {
+      return;
+    }
+    // Look at all commits, and all files changed during those commits.
+    // If they contain a `sync-repo-settings.yaml`, re-sync the repo.
+    function includesConfig() {
+      const fileChangedLists = ['added', 'modified', 'removed'];
+      for (const commit of context.payload.commits) {
+        for (const list of fileChangedLists) {
+          if (commit[list]) {
+            for (const file of commit[list]) {
+              if (file?.includes('sync-repo-settings.yaml')) {
```

Comment:
Should this be a constant to avoid typos?

Reply:
Great call!  Turns out it already existed, and I just had to, you know, like, use it